ReleaseChecklist.md

```diff
@@ -20,50 +20,52 @@
 
 ### Create the Release
  - [ ] Create Github release page: https://github.com/ethereum/solidity/releases/new
- - [ ] On the release page, select the ``release`` branch as new target and set tag to the new version (e.g. `v0.5.4`) (make sure you only `SAVE DRAFT` instead of `PUBLISH RELEASE` before the actual release)
- - [ ] Thank voluntary contributors in the Github release page (use ``git shortlog -s -n -e origin/release..origin/develop``).
- - [ ] Create a pull request from ``develop`` to ``release``, wait for the tests, then merge it.
+ - [ ] On the release page, select the ``develop`` branch as new target and set tag to the new version (e.g. `v0.5.4`) (make sure you only `SAVE DRAFT` instead of `PUBLISH RELEASE` before the actual release)
+ - [ ] Thank voluntary contributors in the Github release page (use ``git shortlog -s -n -e v0.5.4..origin/develop``).
```

Comment:
Maybe we should use the placeholder here too to avoid someone blindly using 0.5.4?
```suggestion
 - [ ] Thank voluntary contributors in the Github release page (use ``git shortlog -s -n -e v$VERSION..origin/develop``).
```

Reply:
should be "previous version"

Reply:
Ah, right. My mistake. So `$PREVIOUS_VERSION`?

Reply:
But in that case  `v0.5.4` was a mistake too. Right above it says "set tag to the new version (e.g. `v0.5.4`)"

Reply:
Right! Will change to v0.6.3